convex/argumentsValidation.ts

```diff
@@ -5,6 +5,8 @@ export const queryWithArgs = query({
   args: {
     a: v.number(),
   },
+  // Never gets called in the test
+  /* v8 ignore next */
```

Comment:
what are these for?

edit: oh based on things below sounds like this is related to coverage?